### PR TITLE
Top instructions metrics

### PR DESCRIPTION
**Pull request by dmcavoy**
Adding or updating firehose events to track usage of different parts of TopInstructions.

1. Clicks on "For Teachers Only" Tab
2. When the instructions are expanded and collapsed and if they are in CSF style instructions or not
3. TTS run button clicked - keep track of which type of instructions we were in
4. Clicks on the "Help & Tips" Tab

### CODE REVIEW

File: apps/src/templates/instructions/TopInstructions.jsx [hunk 1]
```diff
@@ -404,6 +423,10 @@ class TopInstructions extends Component {
 
   handleTeacherOnlyTabClick = () => {
     this.setState({tabSelected: TabType.TEACHER_ONLY}, this.scrollToTopOfTab);
+    firehoseClient.putRecord({
+      study: 'teacher-only-tab',
+      event: 'click'
+    });
```
Comment (Erin007) [inline apps/src/templates/instructions/TopInstructions.jsx:433]: We might want this info more tightly connected in the events table.  Maybe the study name should be consistent across all three?  Something like `top-instructions` and then the events could be `teacher-only-tab-click`, `collapse-instructions`,  and `expand-instructions`.

Comment (dmcavoy) [reply apps/src/templates/instructions/TopInstructions.jsx:433]: Sounds good to me. I don't know how we normally set these things up.

**Review by Erin007 — APPROVED** [review]

File: apps/src/templates/instructions/TopInstructions.jsx [hunk 2]
```diff
@@ -359,6 +360,24 @@ class TopInstructions extends Component {
    * updating our rendered height.
    */
   handleClickCollapser = () => {
+    if (!this.props.collapsed) {
```
Comment (maddiedierker) [inline apps/src/templates/instructions/TopInstructions.jsx]: nit: this conditional statement will be more readable if it's inverted:
```
if (this.props.collapsed) {
  ...
} else {
  ...
}
```

**Review by maddiedierker — APPROVED** [review]
yay, all the metrics!

**Pull request merged**